configs/_base_/models/retinanet_r50_fpn.py

```diff
@@ -66,3 +66,27 @@
         score_thr=0.05,
         nms=dict(type='nms', iou_threshold=0.5),
         max_per_img=100))
+
+tta_model = dict(
```

Comment:
Are the TTA results aligned with mmdet 2.x?

Reply:
Yes, the results have aligned with mmdet 2.26.0.